app/assets/javascripts/respondent_detail.es6

```diff
@@ -0,0 +1,31 @@
+$(document).ready(() => {
+  if ($("#bail_conditions_set").length) {
+    const bailConditionOptions = "input[type = radio][name = 'respondent[bail_conditions_set]']";
+    $(bailConditionOptions).on("change", setBailConditionsLabel);
+    setBailConditionsLabel();
+    if($('#bail_conditions_set_details-error').length) updateError();
+  };
+
+  function setBailConditionsLabel() {
+    const bailConditionsLabel = "label[for='bail_conditions_set_details']";
+    $(bailConditionsLabel).text($('#bail_conditions_set_details').attr(`data-bail-conditions-${selectedBailOption()}`));
+  }
+
+  function selectedBailOption(){
+    let optionChecked = "input[type = radio][name = 'respondent[bail_conditions_set]']:checked";
+    let optionValue = $(optionChecked).val() == 'false' ? 'no' : 'yes'
```

Comment:
you can use `const` here instead of `let` because these variables are not changed